docs/configuration/alertmanager-integration/_alertmanager-config.rst

```diff
@@ -1,28 +1,69 @@
 .. admonition:: AlertManager config for sending alerts to Robusta
 
-    .. code-block:: yaml
-
-        receivers:
-          - name: 'robusta'
-            webhook_configs:
-              - url: 'http://robusta-runner.default.svc.cluster.local/api/alerts' # (2)
-                sendResolved: true # (4)
-
-        route: # (1)
-          routes:
-            - receiver: 'robusta'
-              group_by: [ '...' ]
-              group_wait: 1s
-              group_interval: 1s
-              matchers:
-                - severity =~ ".*"
-              repeat_interval: 4h
-              continue: true # (3)
-
-    .. code-annotations::
-      1. Put Robusta's route as the first route, to guarantee it receives alerts. If you can't do so, you must guarantee all previous routes set ``continue: true`` set.
-      2. This assumes Robusta was installed in the ``default`` namespace, using a Helm release named ``robusta``.
-          * If the namespace is ``foobar``, replace ``default`` with ``foobar``
-          * If the Helm release is named ``robert`` then replace ``robusta`` with ``robert``
-      3. Keep sending alerts to receivers defined after Robusta.
-      4. Important, so Robusta knows when alerts are resolved.
+.. tab-set::
+
+    .. tab-item:: kube-prometheus-stack (Prometheus Operator)
+
+      Add the following to your `AlertManager's config Secret <https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/user-guides/alerting.md#managing-alertmanager-configuration>`_
```

Comment:
Can we update the first sentence (not here, what appears above). As you can see in the below screenshot, it's a little repetitive. ("Add the config below to the appropriate AlertManager file" and then "Add the following to your [AlertManager's config Secret")

Maybe just write "AlertManager is configured differently depending on how it was installed. Choose from the options below."

![Screenshot 2024-01-01 at 13 10 38](https://github.com/robusta-dev/robusta/assets/494087/bfffa7d0-ef69-4dea-ab4a-4da1b54896c3)